core-bundle/assets/styles/component/choices.pcss

```
@@ -0,0 +1,200 @@
+    background: var(--form-bg) url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMiIgaGVpZ2h0PSIxMiIgZmlsbD0ibm9uZSIgc3Ryb2tlPSIjNmE2YTZjIiBzdHJva2UtbGluZWNhcD0icm91bmQiIHN0cm9rZS1saW5lam9pbj0icm91bmQiIHN0cm9rZS13aWR0aD0iMS41IiB2aWV3Qm94PSIwIDAgMjQgMjQiPjxwYXRoIGQ9Im02IDkgNiA2IDYtNiIvPjwvc3ZnPgo=") right -16px top 3px no-repeat;
+    background-origin: content-box;
+    cursor: pointer;
+    text-align: left;
+
+    &:dir(rtl) {
+        text-align: right;
+    }
+
+    &[disabled] {
+        color: var(--form-text-disabled);
+        background-color: var(--form-bg-disabled);
+        border: 1px solid var(--form-border-disabled);
+        cursor: not-allowed;
+    }
+
+    &:focus {
+        outline: revert;
+    }
+
+    &.is-focused {
+        z-index: 4; /* Higher z-index than tinymce and sticky-footer */
+    }
+
+    .choices__inner {
+        background-color: transparent;
+        padding: 0;
+        border: 0 none;
+        border-radius: revert;
+        min-height: unset;
+    }
+
+    .choices__list {
+        &:has(> .choices__group) > .choices__item {
+            padding-left: 20px;
+        }
+
+        &[data-placeholder]:before {
+            content: attr(data-placeholder);
+            display: block;
+            cursor: default;
+            opacity: .7;
+            padding: 3px 6px;
+            font-size: 13px;
+        }
+    }
+
+    .choices__input {
+        background: transparent;
+        padding: 3px 6px;
+        margin: 0;
+        width: auto;
+    }
+
+    .choices__placeholder {
+        opacity: 1;
+    }
+
+    .choices__heading {
+        color: var(--text, #222);
+        cursor: default;
+        border: 0 none;
+        padding: 3px 0 3px 6px;
+        font-size: revert;
+    }
+
+    .choices__button {
+        background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjEiIGhlaWdodD0iMjEiIHZpZXdCb3g9IjAgMCAyMSAyMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48ZyBmaWxsPSIjMjIyIiBmaWxsLXJ1bGU9ImV2ZW5vZGQiPjxwYXRoIGQ9Ik0yLjU5Mi4wNDRsMTguMzY0IDE4LjM2NC0yLjU0OCAyLjU0OEwuMDQ0IDIuNTkyeiIvPjxwYXRoIGQ9Ik0wIDE4LjM2NEwxOC4zNjQgMGwyLjU0OCAyLjU0OEwyLjU0OCAyMC45MTJ6Ii8+PC9nPjwvc3ZnPg==");
+        padding-bottom: 2px;
+        border-left-color: var(--select-border) !important;
+    }
+
+    .choices__list--dropdown, choices__list[aria-expanded] {
+        background: var(--form-bg, #fff);
+        border: 1px solid var(--form-border, #aaa);
+        word-break: break-word;
+        left: -1px;
+
+        .choices__item, .choices__list:before {
+            padding: 3px 6px;
+        }
+
+        .choices__item--selectable,
+        .choices__list[aria-expanded] .choices__item--selectable {
+            &.is-highlighted {
+                background: var(--select-highlighted);
+                color: #fff;
+            }
+        }
+
+        &.is-open {
+            border: 1px solid var(--form-border, #aaa);
+            z-index: 3;
+        }
+    }
+
```

```suggestion
    }

    .is-open {
        .choices__list--dropdown, .choices__list[aria-expanded] {
            border: 1px solid var(--form-border, #aaa);
            z-index: 3;
        }
    }
```
The `is-open` class is on the outer element

---

Right. I don't think we need to set the border at all. I moved it up to where we fix the z-index for the focused state. IMO these can just be both set at 4 (correct me if I'm wrong).

See 087b59b91ae83fdeec79dc4714b07a6565a7c032